certification/internal/policy/operator/scorecard_olm_suite_test.go

```diff
@@ -88,7 +88,7 @@ var _ = Describe("ScorecardBasicCheck", func() {
 			})
 			It("Should not pass Validate", func() {
 				ok, err := scorecardOlmSuiteCheck.Validate(context.TODO(), certification.ImageReference{ImageURI: "dummy/image"})
-				Expect(err).ToNot(HaveOccurred())
+				Expect(err).To(HaveOccurred())
```

Comment:
Same question as the other scorecard test.

Reply:
It's effectively the same.